File: PRAKTIKUM_5/PRAK505-2410817210004-Rahmad.py
def biodata(tahunlahir, nama, asal):
    tahun_sekarang=2020
    usia=tahun_sekarang-tahunlahir
    print("perkenalkan nama saya : ",nama)
    print("Umur saya : ",usia,"tahun")
    print("Saya adalah angkatan  : ",tahun_sekarang)
    print("Asal saya dari : ",asal)
tahunlahir=int(input())
A=str(input())
B=str(input())
biodata(tahunlahir, A, B)

File: PRAKTIKUM_5/PRAK505-2410817210004-Rahmad.c
#include <stdio.h>
void Biodata(int tahunlahir,char nama[],char asal[]){
int tahun_sekarang = 2020;
int usia =tahun_sekarang - tahunlahir;
printf("Perkenalkan Nama saya : %s\n", nama);
printf("Umur saya : %d tahun\n", usia);
printf("saya adalah angkatan : %d\n", tahun_sekarang);
printf("Asal saya dari : %s\n", asal);
// Lengkapi Function ini
}
int main() {
int tahunLahir;
char A[20], B[15];
scanf(" %d",&tahunLahir);
scanf(" %[^\n]%*c",&A);
scanf(" %[^\n]%*c",&B);
Biodata(tahunLahir, A, B);
return 0;
}
